doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -351,6 +351,15 @@ You can use `curl` to hit your endpoint. You should see a `403` Because this exa
 curl -k -w "%{http_code}" https://$(kubectl get httproute test -n ${gatewayNS} -o=jsonpath='{.spec.hostnames[0]}')
 ```
 
+
+### Opening up the Gateway for other namespaces:
+
+As you have now configured the Gateway, secured it with Kuadrant policies and tested it, you can now open it up for use by other teams in other namespaces:
+
+```bash
+kubectl patch gateway ${gatewayName} -n ${gatewayNS} --type='json' -p='[{"op": "replace", "path": "/spec/listeners/0/allowedRoutes/namespaces/from", "value":"All"}]'
```

Comment:
Wouldnt it be better to include it at original Gateway creation section?

Reply:
The original gateway limits HTTPRoutes to the same ns as it is not protected. It is only at this point you have applied the policies and tested connectivity and know it is safe for other namespaces to use the Gateway. At least this is the concept it is intended to convey